Ark.Tools.AspNetCore.ApplicationInsights/Ex.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Ark.Tools.AspNetCore.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.ApplicationInsights.SnapshotCollector;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.ApplicationInsights.WindowsServer.Channel.Implementation;
+using System;
+using System.Diagnostics;
+using System.Reflection;
+using Microsoft.Extensions.Options;
+using Microsoft.ApplicationInsights.DependencyCollector;
+using Microsoft.ApplicationInsights.AspNetCore;
+using Ark.Tools.NLog;
+using Ark.Tools.ApplicationInsights;
+
+namespace Ark.Tools.AspNetCore.ApplicationInsights
+{
+    public static partial class Ex
+    {
+        public static IHostBuilder AddWebServiceInsightsForHostedService(this IHostBuilder builder)
```

Comment:
WebServiceInsights what are them? 

this is an obscure naming.

I'd prefer this Ex to be the 'common' part for AspNetCore Host 

AddApplicationInsithsTelemetry*WebHostArk(...)